extensions/wikia/JSVariables/JSVariables.php

```diff
@@ -58,6 +58,10 @@ function wfJSVariablesTopScripts(Array &$vars, &$scripts) {
 		 * End remove
 		 */
 	}
+	$vars['wikiaIsPowerUserAdmin'] = $user->isSpecificPowerUser( \Wikia\PowerUser\PowerUser::TYPE_ADMIN );
+	$vars['wikiaIsPowerUserFrequent'] = $user->isSpecificPowerUser( \Wikia\PowerUser\PowerUser::TYPE_FREQUENT );
+	$vars['wikiaIsPowerUserLifetime'] = $user->isSpecificPowerUser( \Wikia\PowerUser\PowerUser::TYPE_LIFETIME );
+
```

Comment:
Thanks for the notice. I got a green light from @vforge, but it seems reasonable not to add too many :)